hazelcast/src/main/java/com/hazelcast/map/impl/query/AggregationResult.java

```diff
@@ -55,10 +55,14 @@ public AggregationResult(Aggregator aggregator) {
 
     @Override
     public void combine(AggregationResult result) {
+        Collection<Integer> otherPartitionIds = result.getPartitionIds();
+        if (otherPartitionIds == null) {
```

Comment:
this is possible when the other result was created while partitions were in motion